src/components/ens-profile/ProfileSheetHeader.tsx

```diff
@@ -53,6 +53,7 @@ export default function ProfileSheetHeader({
   const { data: profileAddress } = useENSAddress(ensName, {
     enabled: profilesEnabled,
   });
+  const { rainbowProfile } = useRainbowProfile(profileAddress || '');
```

Comment:
if we start accepting undefined / null values maybe we set enabled in config is the address is defined?

Reply:
actually, we maybe could do that inside the hook, and only set the useQuery call enabled if the address is defined and remove the conditionals to pass a `string` type as we have here and other instances where we use this hook